workload-elsa-data-aws/elsa-data-application/elsa-data-application-construct.ts

```diff
@@ -127,8 +127,6 @@ export class ElsaDataApplicationConstruct extends Construct {
           environment: {
             // we have a DSN that has no password or database name
             EDGEDB_DSN: props.edgeDbDsnNoPassword,
-            // we can choose the database name ourselves or default
-            EDGEDB_DATABASE: props.databaseName ?? "elsadata",
```

Comment:
this allows us to setup new blank databases in demo.. (i.e. if we get the migration state wrong in `elsadata1` we can just make a new blank db `elsadata2` and bump the name here)

(there is a difference between the instances and databases)

Reply:
Ahh that's fair. I think it wasn't working because the default database was being created in the infrastructure repo (e.g. "edgedb"), causing a mismatch. Perhaps it would be best to change change the default name to "edgedb".

Reply:
Its the difference between instances and databases that are the problem. The instance is what is set up via infrastructure (possibly with a blank default). The databases live in the instances and should be named after that app.. i.e. we could have the htsget extension use the same EdgeDb instance and make a database `htsget`